feil-behandler/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/feilbehandler/App.kt

```diff
@@ -34,7 +34,11 @@ fun buildApp(env: Map<String, String>): RapidsConnection {
 
 fun RapidsConnection.createFeilLytter(database: Database): RapidsConnection =
     also {
-        val repository = PostgresBakgrunnsjobbRepository(database.dataSource)
+        createFeilLytter(repository = PostgresBakgrunnsjobbRepository(database.dataSource))
+    }
+
+fun RapidsConnection.createFeilLytter(repository: PostgresBakgrunnsjobbRepository): RapidsConnection =
```

Comment:
Måtte splitte opp denne i to funksjoner for å få TestLytter i integrasjonstestene, siden create-funksjonene ville ha repository som argument der.